Formula/genstats.rb

```diff
@@ -14,8 +18,6 @@ class Genstats < Formula
     sha256 cellar: :any_skip_relocation, el_capitan:    "44502f7a2dfcb1355336db69267d6363d6e8b8767b47628b0d3099743513ed5f"
   end
```

Comment:
```suggestion
  end
  
  deprecate! date: "2022-07-31", because: "Upstream website has disappeared, formula uses archive.org URLs"
```

@SMillerDev Would this suffice?

Reply:
No, it was disabled before. There is no reason for me to change that status.

Reply:
Ah, wouldn't that make it uninstallable?

Reply:
It's already uninstallable. The new url would just make it more convenient to use `brew extract`

Reply:
Hmm. That, versus just... deleting the formula?

Also, should [xml2](https://github.com/homebrew/homebrew-core/blob/master/Formula/xml2.rb) be similarly disabled?

Reply:
```suggestion
  end
  
  disable! date: "2022-07-31", because: "Upstream website has disappeared; formula provides archive.org URLs for 'brew extract' usage"
```

Reply:
> Hmm. That, versus just... deleting the formula?

Deleting it won't warn anyone about the state of the software.

> Also, should [xml2](https://github.com/homebrew/homebrew-core/blob/master/Formula/xml2.rb) be similarly disabled?

If it does not have dependencies in Homebrew it can be deprecated.